### PR TITLE
PTAB search

### PR DESCRIPTION
**Pull request by neka-io**
- Added radio buttons to the search box
- Modified `search.js` to query the selected dataset

To be completed when data is loaded:
- [ ] Verify that result display fields work across datasets 
- [ ] Modify wording where necessary
- [ ] Add fields and conditionals where necessary

### CODE REVIEW

**Comment by bahadasx** [comment]
@noonkay Is this ready to be merged now?

**Comment by neka-io** [comment]
@bahadasx Sure, I can make a different PR for the list above. The functionality is complete. What I would like to make sure is that the search results are displayed with the fields specific to each dataset which is a separate task anyway.

File: searchtool/server/boot/search.js [hunk 1]
```diff
@@ -70,7 +71,8 @@ exports.buildSearch = function (req, res) {
                     pagein:paginate({totalItem:body.response.numFound, itemPerPage:20, currentPage:currentPage, url:'/newsearch',params:{q:q}}),
                     took:humanize.numberFormat(body.responseHeader.QTime,0 ),
                     highlighting:body.highlighting,
-                    term:q
+                    term:q,
+                    ptab: ptab
```
Comment (arowla) [inline searchtool/server/boot/search.js:75]: Can we change this parameter to `dataset`, too? The way it is now is simple for just the two datasets, but will not work once we add more.

**Comment by arowla** [comment]
I had one minor comment. Everything else looks good. Let's try and get this merged and over to Kiran today in preparation for the status meeting.

**Comment by bahadasx** [comment]
@noonkay I think it would be fine either way (if you want to split it out or keep it all in one PR).  The functionality for the PR is to allow someone to search PTAB data and see the results.  So, I could see how seeing the proper fields depending on which dataset you are searching on falling into the same PR (that is assuming it affects how you are able to view the search results when searching PTAB).  I can fork your branch and take a look at it too.

**Comment by bahadasx** [comment]
sounds good @arowla 👍

**Comment by arowla** [comment]
Merging now since Negar is travelling most of today. We'll have to make that `ptab` -> `dataset` change later.

**Pull request merged**